js/src/annotations/mirador21Strategy.js

```diff
@@ -0,0 +1,52 @@
+(function($) {
+
+  $.Mirador21Strategy = function(options) {
+    jQuery.extend(this, {
+      
+    }, options);
+    this.init();
+  };
+  
+  $.Mirador21Strategy.prototype = {
+    init: function() {
+      
+    },
+    
+    isThisType: function(annotation) {
+      if (annotation.on && typeof annotation.on === 'object' &&
+          annotation.on.selector && typeof annotation.on.selector === 'object' &&
+          annotation.on.selector.value && typeof annotation.on.selector.value === 'string' &&
+          annotation.on.selector['@type'] === 'oa:SvgSelector') {
+        return true;
+      }
+      return false;
+    },
+    
+    buildAnnotation: function(options) {
+      var oaAnno = options.annotation,
+          window = options.window,
```

Comment:
Hi, I am not really sure if it is okey to have 'window' declared because it rather ambiguous if it is the global window